odoo/addons/base/static/src/js/res_config_settings.js

```diff
@@ -320,17 +320,19 @@ var BaseSettingRenderer = FormRenderer.extend({
             module.settingView.find('h2').addClass('o_hidden');
             module.settingView.find('.settingSearchHeader').addClass('o_hidden');
             module.settingView.find('.o_settings_container').removeClass('mt16');
-            var resultSetting = module.settingView.find(".o_form_label:containsTextLike('" + self.searchText + "')");
-            if (resultSetting.length > 0) {
-                resultSetting.each(function () {
-                    var settingBox = $(this).closest('.o_setting_box');
+
+            const resultSetting = module.settingView.find(".o_form_label")
```

Comment:
The rest of the file doesn't use ES6 so I'm a bit wary of introducing this.

Reply:
IIRC ES6 is used in the backend in 13.0 and evergreen after that